### ISSUE TITLE
Feature request: Redirect TALP output to file

### ISSUE THREAD

**Issue by jgraciahlrs** [bsc-pm/dlb#4]
TALP reports are currently written to STDOUT (presumably). In many cases this adds noise and may disturb existing workflows which expect certain data in certain positions of the output. 

I suggest to add an option to redirect TALP output to a file. (Personally, I like mpiP's approach to construct the filename.)

**Comment by vlopezh** [comment]
This feature is already implemented, although not yet documented. Our documentation is lagging behind lately due to lack of time.

You can see the flag executing `dlb -h`:
```
DLB is configured by setting these flags in the DLB_ARGS environment variable.
Possible options are listed below:

Option                  Current value   Possible value (type) / [choice] / {val1:val2}
--------------------------------------------------------------------------------------
...
--talp-output-file:			(path)
```

Or in the extended help `dlb -hh`:
```
--talp-output-file:			(path)
    Write TALP metrics to a file. If this option is not provided,
    the output is printed to stderr.
    Accepted formats: *.json, *.xml, *.csv. Any other for plain text.
```

Let us know if this flag covers your needs.

**Comment by jgraciahlrs** [comment]
Cool! This seems to be exactly what I am looking for.

This does not seem to be available in v3.2, though. I guess it is on the main branch. Please confirm and I will test it.

**Comment by vlopezh** [comment]
Yes, this was not available in the last release, but it is in the main branch.
